UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/MetaDataSystem.cs

```diff
@@ -211,11 +211,11 @@ private void SetupNeighbors(MetaDataNode node)
 
 			if (metaTileMap.IsSpaceAt(neighbor, true))
 			{
-				// if current node is a room, but the neighboring is a space tile, this node needs to be checked regularly for changes by other matrices
-				if (node.IsRoom && !externalNodes.ContainsKey(node))
+				/*// if current node is a room, but the neighboring is a space tile, this node needs to be checked regularly for changes by other matrices
+				if (node.IsRoom && !externalNodes.ContainsKey(node) && metaTileMap.IsSpaceAt(node.Position, true) == false)
```

Comment:
This supposed to be commented out?

Reply:
I was having lag issues with the explosion, Turns out if I commenting this out Solve the lag issue And it seems to not impact anything